contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/formulas/MarkdownFormula.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+
+  <span :class="{editing: editing}">
+    <span ref="mathField">
+      <slot></slot>
+    </span>
+  </span>
+
+</template>
+
+<script>
+
+  import Vue from 'vue';
+  import vueCustomElement from 'vue-custom-element';
+  import '../../mathquill/mathquill.js';
+  import css from '!css-loader!less-loader!./style.less';
+
+  const MarkdownFormula = {
+    name: 'MarkdownFormula',
+    props: {
+      editing: {
+        type: Boolean,
+      },
+    },
+    data: {
+      mathquill: null,
+    },
+    mounted() {
+      this.$root.$el.setAttribute('contenteditable', false);
```

Comment:
This is necessary to prevent Squire.js/TUI.editor from letting FireFox delete stuff that's inside of the custom element's slot.